### DISCUSSION TITLE
Package.Json

### DISCUSSION THREAD

**Discussion by itsPrajapat** [storybookjs/storybook#18996] [Help] [answered]
How to install node modules folder as package.json file is not there in project. Please help

**Comment by zhyd1997** [comment, marked as answer]
```shell
# fork and then clone your fork repo.
git clone https://github.com/your-username/storybook.git

# open a terminal at the root folder (terminal 1)
cd scripts && yarn install

# open another terminal at the root folder (terminal 2)
git checkout -b my-first-storybook-contribution
cd code && yarn install
yarn bootstrap
```

**Comment by itsPrajapat** [comment]
Thanks for your reply sir and please let me know sir how to start the
project means where I should run the npm start command.
Thanks & Regards

**Comment by itsPrajapat** [comment]
Dear Sir,
I want to start with open source with this project. Could you please help
me understanding this project so that I could start with.
Thank you
